crates/uv/tests/lock.rs

```diff
@@ -2072,14 +2072,22 @@ fn lock_requires_python() -> Result<()> {
 
     // Install from the lockfile.
     uv_snapshot!(context.filters(), context.sync(), @r###"
-    success: false
-    exit_code: 2
+    success: true
+    exit_code: 0
     ----- stdout -----
 
     ----- stderr -----
     warning: `uv sync` is experimental and may change without warning.
     Removing virtual environment at: [VENV]/
-    error: Requested Python executable `>=3.12` not found in PATH
+    Using Python 3.12.3 interpreter at: /opt/homebrew/opt/python@3.12/bin/python3.12
```

Comment:
This is problematic, need to add a filter for this

Reply:
https://github.com/astral-sh/uv/blob/90947a933df2fe3f42bc9cbbf27dbce3a3b3a708/crates/uv/src/commands/project/mod.rs#L151-L156